### PR TITLE
Ignore division if missing

### PR DESCRIPTION
**Pull request by Jassob**
Not all reports specify a division and this should therefore not be a
required field by the template.

Omitting division previosly caused LaTeX fail with inexplicable
errors--it attempts to end a line and place some vertical space, but
unless the division exists the line is empty and TeX becomes sad.
This commit fixes this by not attempting to typeset division if omitted.

### CODE REVIEW

File: frontmatter/titlepage.tex [hunk 1]
```diff
@@ -77,8 +77,9 @@
 	\end{figure}	\vspace{5mm}
 
 	Department of Computer Science and Engineering\\
+        \ifdefined\thedivision
```
Comment (ayberkt) [inline frontmatter/titlepage.tex]: Is this indentation intended?

**Review by ayberkt — DISMISSED** [review]

File: frontmatter/titlepage.tex (hunk 1 above)
Comment (Jassob) [reply frontmatter/titlepage.tex]: Huh no, it is not... Good catch!

Comment (Jassob) [reply frontmatter/titlepage.tex]: Ugh, it's a tabs-vs-spaces thing...
For some reason my editor does not respect the original tabs and instead indents it with spaces..

**Comment by Jassob** [comment]
@ayberkt I've fixed the indentation now (also added a commit to fix the indentation regarding subtitles). Please re-review :)

File: frontmatter/titlepage.tex (hunk 1 above)
Comment (ayberkt) [reply frontmatter/titlepage.tex]: What? Are these files indented with tabs :angry:? I've never noticed that. I hate tabs and propose that we get rid of all of them.

**Review by ayberkt — APPROVED** [review]

**Pull request merged**